### PR TITLE
Change CPSRelatedContent html to section with role of region

### LINKED ISSUE #5516: a11y P3 - Related Content on PGL pages shouldn't have complimentary region
**Describe the bug**

The related content component at the bottom of the page - Should not be complimentary region - This should be an HTML section with a role of 'region' that is labelled by the components h2. 

Note this is just for this asset type, other asset types may have different requirements such as some of the Media pages. Would be good to check that this is correct elsewhere we are using this too. Discuss with Media team if they want this to change for Media pages or stay the same.

**To Reproduce**
Steps to reproduce the behaviour:

1. Navigate to Related Content with any Screen Reader.

**Expected behaviour**
It should be an HTML section with a role of 'region' that is labelled by the components h2. 

**Screenshots**

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
`<aside>` take a look at the MDN documentation to see what the behaviour of `role` has: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside.

### PR DESCRIPTION
**Pull request by andrewscfc**
Resolves #5516

**Overall change:** 
Adjust HTML markup to inform screen reader that the 'Related Content' section is not a continutation of the main body content but is only loosely related: https://github.com/bbc/simorgh/issues/5516#issuecomment-585294649 

**Note:** This builds upon https://github.com/bbc/simorgh/issues/4807 and has been discussed with @greenc05 

**Code changes:**
- Change wrapping element for Related Content to a `section` with a role of `region`
- Update snapshots and tests to check for new markup

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by greenc05 — APPROVED** [review]
Awesome thank you

File: src/app/pages/MediaAssetPage/index.test.jsx [hunk 1]
```diff
@@ -292,7 +292,7 @@ describe('Media Asset Page', () => {
 
   it('has a single "main" element, and a single "complementary" element (a11y)', async () => {
```
Comment (ryanmccombe) [inline src/app/pages/MediaAssetPage/index.test.jsx]:
```suggestion
  it('has a single "main" element, and a single "region" element (a11y)', async () => {
```

Comment (ryanmccombe) [inline src/app/pages/MediaAssetPage/index.test.jsx]:
```suggestion
  it('has a single "main" element, and a single "region" element (a11y)', async () => {
```

**Review by ryanmccombe — APPROVED** [review]
👍

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**